docs/consumer/Receiver.md

```diff
@@ -1,6 +1,81 @@
 # Stake Receiver
 
 The Stake Receiver is on the consumer side and connected an External Staker on the Provider side.
-This handles the normalization of the token streams and communicates with the Meta-Staker.
+This handles the normalization of the token streams and communicates with "Virtual Staking".
+It is connected to the Provider chain via IBC and "receives" the various staking commands
+
+**TODO** This was called the "Consumer contract" in previous versions and probably needs a better name
+
+## Setup
+
+When we [deploy the contracts](../ibc/Overview.md#deployment), we connect the Stake Receiver on the consumer chain
+with an [External Staking](../provider/ExternalStaking.md) contract on the Provider. Once this connection is established,
+Consumer governance can authorize this Stake Receiver with some ability to mint on the ["Virtual Staking" contract](./VirtualStaking.md).
+
+Note that when we deploy the Stake Receiver, we configure the address of the "virtual staking" contract. We must also
+define a price oracle contract on setup. (see ["Price Normalization"](#price-normalization) below)
+
+## Staking Flow
+
+Once the connection is established, the provider can send various "virtual stake" messages to the receiver, which is responsible
+for processing them and normalizing for the local "virtual staking" module.
```

Comment:
Worth noting that this happens over a dedicated IBC channel and that the receiver contract will only accept messages from its corresponding provider.